e2e/playwright/authentication/src/tests/login.spec.ts

```diff
@@ -0,0 +1,105 @@
+/*!
+ * Copyright © 2005-2023 Hyland Software, Inc. and its affiliates. All rights reserved.
+ *
+ * Alfresco Example Content Application
+ *
+ * This file is part of the Alfresco Example Content Application.
+ * If the software was purchased under a paid Alfresco license, the terms of
+ * the paid license agreement will prevail. Otherwise, the software is
+ * provided under the following open source license terms:
+ *
+ * The Alfresco Example Content Application is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * The Alfresco Example Content Application is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * from Hyland Software. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { expect } from '@playwright/test';
+import { ApiClientFactory, Utils, test } from '@alfresco/playwright-shared';
+
+test.describe('viewer file', () => {
+  const apiClientFactory = new ApiClientFactory();
+
+  const otherLanguageUser = {
+    /* cspell:disable-next-line */
+    username: `пользвате${Utils.random()}`,
+    password: '密碼中國'
+  };
+
+  const johnDoe = {
+    username: `user-${Utils.random()}`,
+    get password() {
+      return this.username;
+    },
+    firstName: 'John',
+    lastName: 'Doe'
+  };
+
+  const testUser2 = {
+    username: `user-${Utils.random()}`,
+    password: 'user2 password'
+  };
+  const newPassword = 'new password';
+
+  test.beforeAll(async () => {
+    await apiClientFactory.setUpAcaBackend('admin');
+
+    await apiClientFactory.createUser(otherLanguageUser);
+    await apiClientFactory.createUser(johnDoe);
+    await apiClientFactory.createUser(testUser2);
+  });
+
+  test.describe('general tests', () => {
+    test('[C213089] login page layout', async ({ loginPage }) => {
+      await loginPage.navigate();
+      expect(await loginPage.username.isEnabled(), 'username input is not enabled').toBe(true);
+      expect(await loginPage.password.isEnabled(), 'password input is not enabled').toBe(true);
+      expect(await loginPage.submitButton.isEnabled(), 'SIGN IN button is enabled').toBe(false);
+      await loginPage.password.fill('text');
+      expect(await loginPage.isPasswordDisplayed(), 'Password is not hidden by default').toBe(false);
+      await loginPage.passwordVisibility.click();
+      expect(await loginPage.isPasswordDisplayed(), 'Password is not hidden by default').toBe(true);
```

Comment:
Small thing, but I think error message on test fail here is not correct, should be '_Password is not visible_' instead of _hidden_?